docs/knowledge_base/phylum_project_files.md

```diff
@@ -0,0 +1,45 @@
+# Phylum Project Files
+
+The `.phylum_project` file is a configuration file containing information about, you guessed it, the Phylum project at that level of the filesystem hierarchy. There is usually a single `.phylum_project` file for a given repository, located at the root of the repository. It is also possible to have multiple `.phylum_project` files within a repository. This may be desirable for monorepos.
```

Comment:
```suggestion
The `.phylum_project` file is a configuration file containing information about the Phylum project at that level of the filesystem hierarchy. There is usually a single `.phylum_project` file for a given repository, located at the root of the repository. It is also possible to have multiple `.phylum_project` files within a repository. This may be desirable for monorepos.
```
I feel like this blurb goes against the tone of the rest of the documentation here.

Reply:
Fair point. It will be updated.